neo/framework/Common.cpp

```diff
@@ -1584,16 +1584,17 @@ void idCommonLocal::Shutdown()
 	printf( "declManager->Shutdown();\n" );
 	declManager->Shutdown();
 
+	// shut down the renderSystem
+	// SRS - Note this also shuts down any testVideo resources, including cinematic audio voices
+	printf( "renderSystem->Shutdown();\n" );
+	renderSystem->Shutdown();
+
 	// shut down the sound system
-	// SRS - Shut down sound system after decl manager so cinematic audio voices are destroyed first
+	// SRS - Shut down sound system after decl manager and render system so cinematic audio voices are destroyed first
 	// Important for XAudio2 where the mastering voice cannot be destroyed if any other voices exist
 	printf( "soundSystem->Shutdown();\n" );
 	soundSystem->Shutdown();
 
-	// shut down the renderSystem
-	printf( "renderSystem->Shutdown();\n" );
-	renderSystem->Shutdown();
-
```

Comment:
Q: Why do we move this block up?

Reply:
_Q: Why do we move this block up?_

This is to prevent a fault on exit when using XAudio2.  If a testVideo containing audio is running during exit, we first need to shutdown that testVideo (with audio) before attempting to shutdown the sound system.  Otherwise XAudio2 cannot complete the `pMasterVoice->DestroyVoice()` action in `soundSystem->Shutdown()` and raises a fault.  There is new code in `renderSystem->Shutdown()` that makes sure the testVideo is deleted, and must be called before `soundSystem->Shutdown()`:

```
// SRS - if testVideo is currently playing, make sure cinematic is deleted before ShutdownCinematic()
if( tr.testVideo )
{
	delete tr.testVideo;
	tr.testVideo = NULL;
}
```